ADAL/ADAL.xcodeproj/project.pbxproj

```diff
@@ -3562,13 +3576,15 @@
 			baseConfigurationReference = D6CF4EB01FC370BF00CD70C5 /* adal__framework__ios.xcconfig */;
 			buildSettings = {
 				GCC_OPTIMIZATION_LEVEL = 0;
+				GCC_PREPROCESSOR_DEFINITIONS = "$(inherited)";
```

Comment:
should this go to xconfig instead?

Reply:
It used to indicate DEBUG=1, I think it should be fine as is.